src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -104,4 +105,126 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->where( 'id', $attendee->ID )
+			              ->set( 'ticket_id', $ticket->ID )
+						  ->set( 'full_name', $attendee_data['full_name'] )
```

Comment:
:warning: Found precision alignment of 2 spaces.

<sub>hash: 3969accccbc74ae2031908edb34946b4</sub>